<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="borrowButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>借书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="repayButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>还书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renewButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>续借</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrowRecordButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>借书记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidgetPermary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>821</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>351</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>图书管系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLineEditPermary">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButtonPermary">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>260</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="searchLineEditBorrow">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButtonBorrow">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QTableView" name="bookInformationborrow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>801</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="affirmBottonBorrow">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>530</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>借书</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnButtonBorrow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableView" name="bookInformationRepay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>811</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="returnButtonRepay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="affirmBottonRepay">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>520</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>还书</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTableView" name="bookInformationRenew">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>811</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="returnButtonRenew">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="affirmBottonRenew">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>520</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>续借</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QTableView" name="bookInformationBorrowR">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>811</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="returnBottonBorrowR">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QTableView" name="inDebtInformation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>811</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="paymentInformation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>451</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="paymentAmount">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>320</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>输入缴费金额</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="paymentAmountInput">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>320</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="passwordMoney">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>400</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordMoneyInput">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>400</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="affirmButtonMoney">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>470</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnButtonMoney">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>843</x>
      <y>0</y>
      <width>20</width>
      <height>591</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="moneyBotton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>缴/欠费</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
